packages/core/src/reducers/index.ts

```diff
@@ -19,7 +19,12 @@ export const getData = state => {
   return extractData(state.common);
 };
 
+const fetchTranslation = state => {
```

Comment:
the fetchTranslation method does not fit the fetchTranslation call
I would inline this if possible